versioned_docs/version-8.6/guides/migrating-from-camunda-7/migration-readiness.md

```diff
@@ -66,7 +66,7 @@ public class CreateCustomerInCrmJavaDelegate implements JavaDelegate {
 
 Never cast to Camunda implementation classes, use any ThreadLocal object, or influence the transaction manager in any way. Java delegates should always be stateless and not store any data in their fields.
 
-The resulting delegate can be migrated to a Camunda 8 API, or reused by the adapter provided in [this migration community extension](https://github.com/camunda-community-hub/camunda-7-to-8-migration/).
+The resulting delegate can be migrated to a Camunda 8 API, or reused by the adapter provided in [this migration community extension](https://github.com/camunda-community-hub/camunda-7-to-8-migration/tree/main/camunda-7-adapter).
```

Comment:
I am a bit hesistant here as this might change - but also not a big deal as I agree it is nicer to be pointed to the right place

Reply:
I see, maybe we can create a follow-up issue for you once the code it moved to the "new" place.
wdyt?